lib/improver/tests/utilities/test_temporal.py

```diff
@@ -104,7 +105,7 @@ def test_alternative_units_defined(self):
         dt = 1511308800.0
         result = cycletime_to_number(
             cycletime, time_unit="seconds since 1970-01-01 00:00:00")
-        self.assertAlmostEqual(result, dt)
+        assert_almost_equal(result, dt, decimal=6)
```

Comment:
Perhaps we could add something to the docstring for this unit test to say why this is needed.

Reply:
I've cast the result as an int and added an explanation to the doc string. I've then changed the comparison to an ```assertEqual```.